src/utils/url/getSearchParams.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @param {string} params
+ * @returns {URLSearchParams}
+ */
```

Comment:
do you use some tool for annotation?

Reply:
It is a standard jsdoc, Enabled by default in WebStorm or with plugin for vscode

Reply:
do you plan to use it for each function in the project?

Reply:
Yes!